publisher.py

```diff
@@ -367,7 +383,17 @@ def process_images(self):
                 for tag in image.release_tags:
                     repository = "%s/%s" % (target, image.repository.name)
                     self.tag_image(image, repository, tag)
-                    self.publish_image(target, repository, tag)
+                    retry_count = 1
+                    while retry_count <= self.retry_limit:
+                        self.log.info("Pushing image, try %d of %d", retry_count, self.retry_limit)
+                        try:
+                            self.publish_image(target, repository, tag)
+                        except ImagePushError as e:
+                            self.log.error("Response %s while pushing %s:%s!", e.response, repository, tag)
+                            retry_count = retry_count + 1
+                            continue
+                        return True
```

Comment:
Move this to the `try` block and remove `continue`?

Reply:
changed it a bit.